### PR TITLE
PLAT-542 Rework repository layout

### PR DESCRIPTION
**Pull request by alexander-schmidt**
- Renamed and/or moved most of the code files.
- Updated scripts and documentation.
- Added a bunch of TODOs for upcoming PRs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,34 +47,74 @@ The following things are required to set up a SoftiCAR GitHub Runner:
...
+      - Look up the default password of the `admin` user in `/var/lib/docker/volumes/nexus-data/_data/admin.password`
+      - Navigate to `http://<ip-of-vm>:8081`, and log in with `admin` / `(default-password)`
+      - Perform the displayed initialization steps. **Caution:** The default `admin.password` file will be deleted in the process.
+   1. Configure a **Docker image proxy:**
+      - Head to `Settings` / `Repository` / `Repositories`, click `Create repository`, and select `docker (proxy)`
+      - Enter:
+        - _Name:_ `docker-hub`
+        - _HTTP:_ Enabled, and enter `8123`
+        - _Allow anonymous docker pull:_ Enabled
+        - _Enable Docker V1 API:_ Enabled (**TODO** try without this)
```
Comment (richers) [inline README.md:80]: note: according to the docker docu, v2 does not support searching, if we need that, I cannot tell

File: control.sh [hunk 2]
```diff
@@ -1,39 +1,41 @@
...
-SERVICE_SCRIPT_PATH=$SELF_PATH/softicar-github-runner.sh
-SERVICE_SCRIPT_ENVIRONMENT_FILE=softicar-github-runner.env
+SERVICE_TEMPLATE_PATH=systemd-service/softicar-github-runner.service-template
+SERVICE_SCRIPT_PATH=$SELF_PATH/systemd-service/softicar-github-runner-service.sh
+SERVICE_SCRIPT_ENVIRONMENT_FILE=softicar-github-runner-service.env
 RUNNER_NAME_DEFAULT="softicar-github-runner"
 RUNNER_NAME_REGEX="[a-zA-Z0-9]([_.-]?[a-zA-Z0-9]+)*"
 RUNNER_LABELS_DEFAULT="ephemeral,dind"
-REPOSITORY_NAME_EXAMPLE="SoftiCAR/some-repo"
+REPOSITORY_NAME_EXAMPLE="softicar/some-repo"
```
Comment (richers) [inline control.sh:21]: somehow, this is half-baked, suggestion: either as template `REPOSITORY_NAME_EXAMPLE="<owner>/<repository>` or full example `REPOSITORY_NAME_EXAMPLE="softicar/platform`

**Review by richers — DISMISSED** [review]
fine in general

File: control.sh (hunk 2 above)
Comment (alexander-schmidt) [reply control.sh:21]: going with the former

File: README.md (hunk 1 above)
Comment (alexander-schmidt) [reply README.md:80]: Good to know. We'll see about that in the final testing phase.

**Review by richers — APPROVED** [review]

**Pull request merged**